server/schedule/opts.go

```diff
@@ -1,34 +0,0 @@
-// Copyright 2017 PingCAP, Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package schedule
-
-import (
-	"time"
-)
-
-// Options for schedulers.
-type Options interface {
```

Comment:
maybe we can embed this interface to Cluster, I think Filter/Selector use Options interface more clearly.